### PR TITLE
[627], [628] add showquestioncode, cookieconsent, footerbranding columns

### PR DESCRIPTION
**Pull request by jack-han-115**
Fixed issue #<Mantis issue number>:
New feature #<Mantis issue number>:
Dev:

### CODE REVIEW

File: application/helpers/update/updates/Update_620.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace LimeSurvey\Helpers\Update;
+
+class Update_620 extends DatabaseUpdateBase
+{
+    public function up()
+    {
+            // Add new tokens setting
+            addColumn('{{surveys}}', 'showquestioncode', "string(1) NOT NULL default 'N'");
```
Comment (Shnoulle) [inline application/helpers/update/updates/Update_620.php:10]: ? https://github.com/LimeSurvey/LimeSurvey/blob/4139afef7446f37e8a3bc2dfd9ce2423f8265d8a/installer/create-database.php#L662

File: application/libraries/Api/Command/V1/Transformer/Input/TransformerInputSurvey.php [hunk 2]
```diff
@@ -74,7 +74,11 @@ public function __construct()
             'googleAnalyticsApiKey' => 'googleanalyticsapikey',
             'showSurveyPolicyNotice' => ['key' => 'showsurveypolicynotice', 'type' => 'int'],
             'template' => true,
-            'format' => true
+            'format' => true,
+            // below fields are newly added
```
Comment (kevin-foster-uk) [inline application/libraries/Api/Command/V1/Transformer/Input/TransformerInputSurvey.php:78]: This comment is not needed.

File: application/libraries/Api/Command/V1/Transformer/Output/TransformerOutputSurvey.php [hunk 3]
```diff
@@ -79,7 +79,12 @@ public function __construct(
             "googleanalyticsapikey" => 'googleAnalyticsApiKey',
             "showsurveypolicynotice" => ['key' =>  'showSurveyPolicyNotice', 'type' => 'int'],
             'template' => true,
-            'format' => true
+            'format' => true,
+            // below fields are newly added
```
Comment (kevin-foster-uk) [inline application/libraries/Api/Command/V1/Transformer/Output/TransformerOutputSurvey.php:83]: Comment not needed here.

File: application/models/services/SurveyAggregateService/GeneralSettings.php [hunk 4]
```diff
@@ -268,7 +267,11 @@ private function getFields(Survey $survey)
             'bounce_email' => [],
             'gsid' => ['default' => 1],
             'format' => [],
-            'template' => []
+            'template' => [],
+            // below fields are newly added
```
Comment (kevin-foster-uk) [inline application/models/services/SurveyAggregateService/GeneralSettings.php:271]: Comment not needed.

File: installer/create-database.php [hunk 5]
```diff
@@ -676,6 +676,11 @@ function populateDatabase($oDB)
             'googleanalyticsstyle' => "string(1) NULL",
             'googleanalyticsapikey' => "string(25) NULL",
             'tokenencryptionoptions' => "text NULL",
+            // these are newly added columns
```
Comment (kevin-foster-uk) [inline installer/create-database.php:679]: Comment not needed.

**Comment by kevin-foster-uk** [comment]
Just the remove the unnecessary comments. All good, other than that.

**Pull request merged**